pkgs/by-name/aw/await/package.nix

```diff
@@ -3,20 +3,24 @@
   stdenv,
   fetchFromGitHub,
   installShellFiles,
+  versionCheckHook,
 }:
 
 stdenv.mkDerivation rec {
   pname = "await";
-  version = "1.0.2";
+  version = "1.0.5";
 
   src = fetchFromGitHub {
     owner = "slavaGanzin";
     repo = "await";
     rev = "v${version}";
-    hash = "sha256-qvSRuRLZnUptXYknyRn4GgmYtj9BnI8flN6EhadbKMw=";
+    hash = "sha256-0U9eLQDvHnRUJt46AI4bDWZfGynqjaWs9teidWP3RsA=";
   };
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [
+    installShellFiles
+    versionCheckHook
```

Comment:
`versionCheckHook` should be in `nativeInstallCheckInputs`, please see `doc/hooks/versionCheckHook.section.md`